test/clojure/core_test/constantly.cljc

```diff
@@ -0,0 +1,44 @@
+(ns clojure.core-test.constantly
+  (:require clojure.core
+            [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer)  [when-var-exists]]))
+
+(when-var-exists clojure.core/constantly
+  (deftest test-constantly
+    (testing "`constantly`"
+      (is (fn? (constantly nil)))
+
+      (are [v] (= v ((constantly v)))
+        `sym
+        "abc"
+        100000
+        1.0
+        #?(:clj 2/3) ; ratio not supported in cljs
+        \return
+        nil
+        true
+        false
+        :keyword
+        :namespace/keyword
+        '(one two three)
+        [4 5 6]
+        {:7 "8"}
+        (zipmap (take 1000 (range))
```

Comment:
Get what you're going for but 1000 may be overkill, I think 100 would probably be plenty, I know this doesn't make a noticeable difference in performance but if everyone adds a 1k range to each test then it probably will accumulate to a noticeable increase in test execution time